static/app/components/analyticsAreaProvider.tsx

```diff
@@ -0,0 +1,52 @@
+import {createContext, type ReactNode, useContext} from 'react';
+
+export const AnalyticsAreaContext = createContext<string>('');
```

Comment:
How about instead of exporting this you just export `useAnalyticsArea`

```tsx
export function useAnalyticsArea() {
  return useContext(AnalyticsAreaContext);
}
```

Reply:
This is a bit cleaner since you wouldn't ever want anyone to render their own `AnalyticsAreaContext.Provider`